src/wakefs/daemon_client.cpp

```diff
@@ -34,9 +34,16 @@
 #include "json/json5.h"
 #include "fuse.h"
 
+// The user-id and group-id are used so that fuse daemons with different uid:gid pairs
+// running within the same build can co-exist without trying to share. The kernel
+// prevents cross-user sharing when the 'allow_other' fuse mount option is not used.
+const std::string mount_path_id(const std::string& base_dir) {
+    return base_dir + "/.fuse-" + std::to_string(getuid()) + ":" + std::to_string(getgid());
+}
```

Comment:
I didn't understand yesterday but I now get why you guys were laughing. This is painfully simple lol.